README.md

```diff
@@ -70,8 +70,8 @@ spin up.
 You now have set up Kong as your Ingress controller and
 all Ingress resources in your Kubernetes Cluster will be satisfied.
 
-Please refer our [deployment documentation][deployment-doc]
-for a detailed  introduction to Kong Ingress Controller
```

Comment:
I think "a detailed introduction" is correct grammar.
@coopr ?

Reply:
Yep, "Please refer our deployment documentation for a detailed introduction to Kong Ingress Controller" is certainly correct.

There is, however, an extra space between "detailed" and "introduction" that could be removed :)